### ISSUE TITLE
[Reanimated] Couldn't determine the version of the native part of Reanimated.

### ISSUE THREAD

**Issue by kishieel** [expo/expo#26534]
### Minimal reproducible example

https://github.com/expo/examples/tree/master/with-reanimated

### Summary

As in title, reporoduction:

1. npx create-expo-app@latest my-app -e with-reanimated
2. yarn install
3. yarn expo install
4. yarn remove react-native-web
5. yarn start --clear
6. Open app with Expo GO

### Environment

  expo-env-info 1.0.5 environment info:
    System:
      OS: Linux 6.6 Arch Linux
      Shell: 5.9 - /bin/zsh
    Binaries:
      Node: 18.13.0 - ~/.nvm/versions/node/v18.13.0/bin/node
      Yarn: 1.22.21 - /usr/bin/yarn
      npm: 8.19.3 - ~/.nvm/versions/node/v18.13.0/bin/npm
    IDEs:
      Android Studio: AI-222.4459.24.2221.10121639
    npmPackages:
      expo: ^50.0.1 => 50.0.2 
      react: 18.2.0 => 18.2.0 
      react-dom: 18.2.0 => 18.2.0 
      react-native: 0.73.2 => 0.73.2 
    Expo Workflow: managed

**Comment by brentvatne** [comment]
i added `@expo/metro-runtime` to that example so you don't need to remove `react-native-web` anymore.

i can reproduce the warning

**Comment by hyochan** [comment]
This is also happening to my boilerplate project.
https://github.com/dooboolab-community/expo-router-starter

I did everything but could not resolve this!

**Comment by GoldMyr1994** [comment]
This is happening also to me

**Comment by eybel** [comment]
Also here, I got this message "Couldn't determine the version of the native part of Reanimated.". I am using:

  "dependencies": {
    "@expo/vector-icons": "^14.0.0",
    "@react-navigation/drawer": "^6.6.6",
    "@react-navigation/native": "^6.0.2",
    "expo": "~50.0.2",
    "expo-font": "~11.10.2",
    "expo-linking": "~6.2.2",
    "expo-router": "~3.4.3",
    "expo-splash-screen": "~0.26.3",
    "expo-status-bar": "~1.11.1",
    "expo-system-ui": "~2.9.3",
    "expo-web-browser": "~12.8.1",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "react-native": "0.73.2",
    "react-native-gesture-handler": "~2.14.0",
    "react-native-reanimated": "~3.6.0",
    "react-native-safe-area-context": "4.8.2",
    "react-native-screens": "~3.29.0",
    "react-native-web": "~0.19.6"
}

**Comment by tsapeta** [comment]
I can reproduce the warning. I think https://github.com/software-mansion/react-native-reanimated/pull/5499 may solve it, but it hasn't been released yet. I will follow up on this with the reanimated team.

**Comment by eybel** [comment]
Foun the error. There is a property REANIMATED_VERSION_CPP inside global that is not available. Its undefined then it compares with version 3.5.4 thus the error. The file I fixed "temporary" is inside the react-native-reanimated package

Providing proof:

[IMAGE with fallback to 3.5.4 (by me)](https://ibb.co/HhKVkZY)

**Comment by jeokeefe** [comment]
having same issue... just upgraded from 49 to 50.0.2

**Comment by brentvatne** [comment]
fyi - this is just a warning, and it only appears in Expo Go. you can still use reanimated just fine in Expo Go with SDK 50. we're waiting on a new reanimated release that fixes this warning, and we'll then release new version of Expo Go

- you can ignore the warning ([see logbox docs](https://reactnative.dev/docs/debugging#logbox))
- you can use [development builds](https://docs.expo.dev/develop/development-builds/introduction/) (generally recommended for real world applications)

**Comment by danieldenton** [comment]
This seemed to be crashing my production builds. I still get the error but I fixed the production build crashes by upgrading to Expo 50.0.3.

**Comment by glundgrenm** [comment]
@danieldenton thanks for pointing it! I was going mad here. I confirm that my build was crashing on android with 50.0.0 and after update to 50.0., build with eas build and submitted to Play Console with eas submit, the app did not crashed.

**Comment by HelderBarreto** [comment]
Im having the same warning. I thought it was a fault of mine becuase im new on reazct native,  but I tried to create a new project with reanimated integrated with this command: 

npx create-expo-app@latest SmartChargingMobileApp -e with-reanimated

and I got the error. So we need to wait for the reanimated updates with the bug fixed.

**Comment by brentvatne** [comment]
@danieldenton - this will not crash your debug builds, that was likely something else. if you believe this was caused by reanimated, please create a minimal reproducible example in a new issue. this issue only impacts Expo Go.

**Comment by brentvatne** [comment]
i'm going to lock this issue for now, please refer to https://github.com/expo/expo/issues/26534#issuecomment-1906869521 for context

**Comment by tsapeta** [comment]
`react-native-reanimated@3.6.2` has been released – it shouldn't trigger the warning in Expo Go anymore, can someone try it and confirm?

**Comment by jetaggart** [comment]
This appears to be gone for me as well. Thanks!

**Comment by kishieel** [comment]
I can confirm that the warning is gone with upgraded version. Thanks!

**Comment by aryankatebian** [comment]
I can confirm the fix too thanks

**Comment by eybel** [comment]
FIxed
